src/Microsoft.ML.LightGBM/LightGbmRankingTrainer.cs

```diff
@@ -127,7 +127,11 @@ protected override void CheckAndUpdateParametersBeforeTraining(IChannel ch, Role
     /// </summary>
     public static partial class LightGbm
     {
-        [TlcModule.EntryPoint(Name = "Trainers.LightGbmRanker", Desc = "Train an LightGBM ranking model", UserName = LightGbmRankingTrainer.Summary, ShortName = LightGbmRankingTrainer.ShortName)]
+        [TlcModule.EntryPoint(
+            Name = "Trainers.LightGbmRanker", 
+            Desc = "Train a LightGBM ranking model. This API requires Microsoft.ML.LightGBM nuget.", 
```

Comment:
>requires Microsoft.ML.LightGBM nuget [](start = 61, length = 36)

Entry-points are not just for the C# API. (Indeed, they won't be for the C# API at all.) This description will also show up in the other language bindings. This note is specific to .NET, so it does not belong here.